scripts/oppgave8.js

```diff
@@ -0,0 +1,41 @@
+// Description:
+//   Script for å sjekke resultater fra Eliteserien 2017
+//
+// Commands:
+//   hubot hvem leder eliteserien
+//   hubot hvordan går det med <lag>
+
+const url = "https://api.vglive.no/v1/vg/tournaments/seasons/1439/standings/live";
+
+module.exports = function (robot) {
+  robot.respond(/hvem leder eliteserien/i, function (msg) {
+    robot.http(url).get()(function (err, res, body) {
```

Comment:
Kanskje du kan bruke `request`-biblioteket her i stedet for `robot.http`? Da blir det mer likt den kantine-oppgaven (og for så vidt også yr-oppgaven)